src/drivers/uavcannode/Publishers/GnssFix2.hpp

```diff
@@ -118,6 +120,26 @@ class GnssFix2 :
 			fix2.covariance.push_back(gps.s_variance_m_s);
 			fix2.covariance.push_back(gps.s_variance_m_s);
 
+			uavcan::equipment::gnss::ECEFPositionVelocity ecefpositionvelocity{};
+			ecefpositionvelocity.velocity_xyz[0] = NAN;
+			ecefpositionvelocity.velocity_xyz[1] = NAN;
+			ecefpositionvelocity.velocity_xyz[2] = NAN;
+
+			// Use ecef_position_velocity for now... There is no heading field
```

Comment:
This makes absolutely no sense. Worse than confusing track angle and heading. Only good news is that we don't use ECEF in our current configuration. We should use it for long missions and earth rotation sensitive sensors in the future.